src/runtime/runtime_tinygowasmp2.go

```diff
@@ -0,0 +1,88 @@
+//go:build tinygo.wasm && wasip2
+
+package runtime
+
+import (
+	"unsafe"
+)
+
+type __wasi_io_stream_list struct {
+	buf unsafe.Pointer
+	len uint32
+}
+
+//go:wasmimport wasi:cli/stdout@0.2.0-rc-2023-11-10 get-stdout
```

Comment:
Are these modules stable? NGL I figured there'd be something like `wasi_snapshot_preview2:cli/stdout` maybe, not this 😅.

Reply:
According to @ricochet, if a component expresses a version number, then the host must support that exact version.

Reply:
The version number stuff will be solved by component registries.

Reply:
This will be `wasi:cli/stdout@0.2.0` once a second reference implementation is complete.